Sources/ViewControllers/Products/Detail/ProductDetailViewController.swift

```diff
@@ -47,7 +47,7 @@ class ProductDetailViewController: ButtonBarPagerTabStripViewController, DataMan
         }
         setUserAgent()
 
-        NotificationCenter.default.addObserver(forName: .productChangesUploaded, object: nil, queue: .main) { [weak self] notif in
+        _ = NotificationCenter.default.observe(name: .productChangesUploaded, object: nil, queue: .main) { [weak self] notif in
```

Comment:
If I understand the new method in the helper correctly, the observer would be removed once "deinited", but if we do not keep any reference (by using `_ = `), would'nt the object be 'deinited' soon after ?

Reply:
Thank you @philippeauriach for pointing this! That's true. I fixed that, what we need to do is store the wrapper instance in a private property so that it gets destroyed when its owner (the view controller) gets deallocated. So this is equivalent to manual unregistering in deinit.